x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -52,11 +55,26 @@ export interface UpdateOptions<Params extends AlertTypeParams> {
   index: string;
 }
 
+export interface BulkUpdateOptions<Params extends AlertTypeParams> {
+  ids: string[] | undefined | null;
+  status: 'open' | 'closed';
```

Comment:
I know there's been a lot of discussion around what statuses can be. Haven't followed, but might want to double check that these are indeed the terms we're going with and maybe create the type for it to be shared?

Reply:
The alerts as data spreadsheet defines it as `Must be one of {open, acknowledged, closed}.` so I'll create an exported common type (maybe in the rule registry package where the technical terms are defined) and update that to include `acknowledged`

good catch!